wallet/src/Pos/Wallet/Web/Api.hs

```diff
@@ -111,6 +111,9 @@ instance ReportDecodeError (WalletVerb (Verb (mt :: k1) (st :: Nat) (ct :: [*])
     reportDecodeError _ err = throwM (DecodeError err) ^. from serverHandlerL'
 
 instance ( HasServer (Verb mt st ct $ ApiModifiedRes WalletVerbTag a) ctx
+         , AllCTRender ct a
+         , KnownNat st
+         , ReflectMethod mt
          , Reifies config ApiLoggingConfig
          , ReflectMethod mt
```

Comment:
Duplicated constraint.
Btw, why not just require `HasServer (WalletVerb (Verb mt st ct)) ctx`? AFAIU, these constraints (excluding `Reifies` and `Buildable` ones) are here only to imply this constraint. Am I missing something?

Reply:
Actually, `HasServer (WalletVerb (Verb mt st ct)) ctx` doesn't use logging. Those constraints would be implied by `HasServer (LoggingApi ...) ctx` or `HasLoggingServer ...`.
For some reason they are not enough as well though.

As justification I can generalize this instance and move to `Util.Servant`